eclass/kernel-build.eclass

```diff
@@ -20,6 +20,14 @@
 # the kernel and installing it along with its modules and subset
 # of sources needed to build external modules.
 
+# @ECLASS_VARIABLE: KV_FULL
+# @DESCRIPTION:
+# A string containing the full kernel release version, e.g.
+# '6.9.6-gentoo-dist'. This is used to ensure consistency between the
+# kernel's release version and Gentoo's tooling. If not already set,
+# kernel-build_src_compile() will set this using the built kernel's
+# kernel.release file.
+
```

Comment:
So there's a valid use case for setting it from the ebuild? If so, say it straight.